bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBulkMutation.java

```diff
@@ -33,6 +31,9 @@
    */
   void flush() throws InterruptedException;
 
+  /**
+   * Runs unfinished task. This method blocks until accumulated tasks are finished.
```

Comment:
I think the comment is not clear.

Reply:
Reword it to:

Send any outstanding {@link RowMutation}s, present in the current batch.

Reply:
Sends ?

Reply:
ohh, Yupp `Sends` would be more appropriate here. Thanks